samples/packaging/actively_supported/Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "activesupport"
```

Comment:
it's not  a biggy but... why of all gems would we pull in activesupport? ;P #noloveformonkeypatces 

(doesn't need to be "fixed" of course, just my grumpy self)